### ISSUE TITLE
Fix typos and other errors in NOTICES.md or remove it

### ISSUE THREAD

**Issue by aababilov** [MobilityData/gtfs-validator#943]
Fix typos and other errors in NOTICES.md or remove it. This file is hard to maintain manually because it aggregates all the notices produced by validator.

Some errors that I have found so far:

```
| `routeId`        	| The id of the faulty record.            	| Long   	|
```
routeId must be string, not long.

```
| `csvRowNumber`   	| The row number of the faulty record.    	| String 	|
```
csvRowNumber must be long, not string.

```
| `block_trips_with_overlapping_stopTimes`               	| [`BlockTripsWithOverlappingStopTimesNotice`](#BlockTripsWithOverlappingStopTimesNotice)                         	|
| `decreasing_or_equal_stopTime_distance`                	| [`DecreasingOrEqualStopTimeDistanceNotice`](#DecreasingOrEqualStopTimeDistanceNotice)                           	|
```

Wrong capitalization.

**Comment by isabelle-dr** [comment]
Thank you for opening this issue, the typos you mentioned + some other similar ones have been fixed in #940 

What was the reason for having two files (NOTICES and RULES)? I feel like both files are needed to either: 
1. interpret the validation report 
2. understanding the rules

RULES.md has:
- a quick description (in table), a longer description (below) and the provenance of the rule (python validator, spec...)

NOTICES.md has:
- the notice code as seen in the validation report
- the affected files
- the schema of the results

@lionel-nj do you see any potential issues or confusion to having only one file?

**Comment by lionel-nj** [comment]
> What was the reason for having two files (NOTICES and RULES)?

We initially saw `RULES.md` as a file that would describe the validation rules and `NOTICES.md` as a way to give more information about a specific notice.

> @lionel-nj do you see any potential issues or confusion to having only one file?

Indeed, having information in both files is confusing - I would advocate to only have one: `RULES.md` because since `NOTICES.md` is manually edited (as @aababilov mentioned: this leads to merging conflicts, typos, and possible inaccuracy); the file is hard to maintain.  Also in https://github.com/MobilityData/gtfs-validator/pull/925 there is a discussion to get the information from `NOTICES.md` into the javadocs.
